flytekit/core/container_task.py

```diff
@@ -249,6 +251,11 @@ def execute(self, **kwargs) -> LiteralMap:
         )
         # Wait for the container to finish the task
         # TODO: Add a 'timeout' parameter to control the max wait time for the container to finish the task.
+
+        if self.local_logs:
```

Comment:
Can we add an indication that these logs are coming from the local container?

Reply:
@eapolinario I have made some changes to indicate that logs are coming from the local container.